packages/smooth_app/lib/query/products_preload_helper.dart

```diff
@@ -1,15 +1,14 @@
 import 'dart:io';
 import 'package:openfoodfacts/openfoodfacts.dart';
 import 'package:smooth_app/database/dao_product.dart';
-import 'package:smooth_app/database/local_database.dart';
 import 'package:smooth_app/query/product_query.dart';
 
 class PreloadDataHelper {
-  PreloadDataHelper(this.localDatabase);
-  LocalDatabase localDatabase;
+  PreloadDataHelper(this.daoProduct);
 
-  Future<String> preloadData() async {
-    final DaoProduct daoProduct = DaoProduct(localDatabase);
+  final DaoProduct daoProduct;
+
+  Future<String> getTopProducts() async {
```

Comment:
* I would return an `int?` instead of a `String`, for localization
* I would rename the method - here we download and save top products ("get" is too ambiguous, could be a local field)
* I would add a `///` comment

Reply:
I agree string is kind of not okay for this, probably a better choice would be a status class here but continuing with int for now